### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by RileyManda**
Pull request for review on project setup: Implemented Linters,Created Html view with H1 tag title and added a .gitignore file to the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <h1 class="h1">Hello Microverse!</h1>
+
+</body>
```
Comment (iambenkis) [inline index.html:15]: - [ ] Keep doing the great work 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @RileyManda  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

Capture

- [ ] Good job implementing the `Hello world project ,  everything looks good , however you didn't use the microverse template found here as required. Kindly check this link and update that.
- [ ] Make sure you customize that README template with information relevant to your project 👍  

dddd
- [ ] I notice that a broken link was attached to your MIT license , Kindly avoid broken links to make your file more professional 👍 


Capture

- [ ] Kindly consider writing the commit messages where the first word should be capitalized following the best practices. As you can't change the commit messages now so this is just a reminder for future projects.

## Optional suggestions

- [ ] [OPTIONAL] This note is only instructions for you to set up the license. I think you should remove it to keep your README looking professional.


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (RileyManda) [reply index.html:15]: @iambenkis Thank you for the review and feedback on the project. I will make the necessary changes to the Readme file as requested and will ensure that  all commits follow the recommended format of a capitalized first letter.

**Review by taldr27 — APPROVED** [review]
## Hi @RileyManda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats
## Highlights 🎉
✅ Professional and customized Readme. 
✅ Linters tests working fine.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**